client/public/locales/en/translation.json

```diff
@@ -68,6 +68,7 @@
     "answers": "Answers",
     "browse-list": "Browse the list of statements you have answered so far",
     "delete-account": "Delete account",
+    "signout": "Sign out",
```

Comment:
Can you also add other translations for the ones we have to this?

Reply:
yep, adding right now